src/styles/scss/index.scss

```diff
@@ -1,3 +1,19 @@
-h1 {
-    color: rgb(15, 224, 95);
-}
\ No newline at end of file
+// Variables should be the first because all other files are inheriting from them
+@import "utils/variables";
+
+//IMPORT ALL THE OTHER FILES HERE
+@import "base/resets";
+@import "base/base";
+@import "components/avatars";
+@import "components/badge";
+@import "components/buttons";
+@import "components/cards";
+@import "components/comments";
+@import "components/icons";
+@import "components/nav";
+@import "layout/navbar";
+@import "pages/auth";
+@import "pages/articles";
+@import "utils/helpers";
+@import "pages/profile";
```

Comment:
We have way too many css helpers, that we don't for now. Again, the task here is setting up `react-router`. I don't see how these files are helping us for now. If you would like to have basic styling written, create a chore.

Reply:
okay I will remove them. We had them wired during the mockup and our workflow integrates perfectly with it